src/plugins/intel_npu/tools/protopipe/src/parser/config.cpp

```diff
@@ -192,6 +216,9 @@ struct convert<UniformGenerator::Ptr> {
 template <>
 struct convert<IRandomGenerator::Ptr> {
     static bool decode(const Node& node, IRandomGenerator::Ptr& generator) {
+        std::cout << "Septi: " << __FILE__ << ":" << __LINE__ << "   ---   " << __func__ << std::endl;
+        const std::unordered_set<std::string> parameters = {"dist", "high", "low", "uniform"};
```

Comment:
`uniform` is value and is not the key, please remove it

Reply:
Done